### PR TITLE
bkontur: retain at rank1

### PR DESCRIPTION
**Pull request by bkontur**
[Rendered](https://github.com/bkontur/polkadot-fellows-evaluations/blob/bko-evidence-rank1-2024-11/evidence/bkontur/0002-bkontur-2024-retention-rank1-evidence.md)

### CODE REVIEW

File: evidence/bkontur/0002-bkontur-2024-retention-rank1-evidence.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Evidence-0002: Retention at to Rank 1
```
Comment (anaelleltd) [inline evidence/bkontur/0002-bkontur-2024-retention-rank1-evidence.md:1]: Typo: "Retention at Rank 1"

Comment (bkontur) [inline evidence/bkontur/0002-bkontur-2024-retention-rank1-evidence.md:1]:
```suggestion
# Evidence-0002: Retention at Rank 1
```

Comment (bkontur) [reply evidence/bkontur/0002-bkontur-2024-retention-rank1-evidence.md]: thank you, fixed :)

**Review by anaelleltd — APPROVED** [review]

**Comment by bkontur** [comment]
Hey @anaelleltd, just one small improvements, I think you can use also "Squash and Merge" when merging, then the history of `main` branch would look little bit nicer: 1 commit per evidence instead of branch spaghetti :)

image

**Comment by anaelleltd** [comment]
> I think you can use also "Squash and Merge" when merging, then the history of `main` branch would look little bit nicer: 1 commit per evidence instead of branch spaghetti :)

Oh, thanks for letting me know about this! I had no idea these commits were overlapping!  🙏

**Pull request merged**